data/102/555/091/102555091.geojson

```diff
@@ -19,7 +19,7 @@
     "mz:categories":[],
     "mz:hierarchy_label":1,
     "mz:is_current":1,
-    "mz:min_zoom":13.0,
+    "mz:min_zoom":13,
```

Comment:
Our [zoom](https://github.com/whosonfirst/whosonfirst-properties/blob/master/properties/mz/max_zoom.json) values should remain floats (`13.0`, rather than `13`).

Reply:
The docs were run through 'exportify' so I don't have any control over how they are serialised 🤷